test/morgan.js

```diff
@@ -1117,12 +1188,11 @@ describe('morgan()', function () {
       it('should color 5xx red', function (done) {
         var cb = after(2, function (err, res, line) {
           if (err) return done(err)
-          assert.strictEqual(line.substr(0, 37), '_color_0_GET / _color_31_500_color_0_')
-          assert.strictEqual(line.substr(-9), '_color_0_')
```

Comment:
Why remove this part? This is to test that the end of the message resets the console back so non-morgan `console.log` lines are not suddenly colored.

Reply:
sorry - added back!